doc/api/sqlite.md

```diff
@@ -692,6 +692,17 @@ added:
 By default, if an unknown name is encountered while binding parameters, an
 exception is thrown. This method allows unknown named parameters to be ignored.
 
+### `statement.setReturnArrays(enabled)`
+
+<!-- YAML
+added: REPLACEME
```

Comment:
I'm unsure what to include here. To the best of my knowledge, this was released in `24.0.0`, but I'm unsure whether I should hardcode this value here.

Releasers might have a better idea.

Reply:
Yes, hardcode the version if the documentation was missed. We'll need a manual backport of this PR to all versions the `setReturnArrays` was released too.